exercises/rotational-cipher/rotational_cipher.py

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
There is already a description in the markdown file, so I wouldn't add any doc string here. We tend to keep the stub files very limited, so we don't force any specific way of implementing an exercise or naming parameters onto the users. This should be enough:
```python
def rotate():
    pass
```